docs/en/latest/plugins/limit-conn.md

```diff
@@ -93,17 +99,19 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
-You also can complete the above operation through the web interface, first add a route, then add limit-conn plugin:
-![enable limit-conn plugin](../../../assets/images/plugin/limit-conn-1.png)
+<!-- ![enable limit-conn plugin](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/plugin/limit-conn-1.png) -->
```

Comment:
Do we need to keep this?

Reply:
I have been commenting out images everywhere just so that we know we have these images in the server. Once we decide how to document APISIX dashboard, we can decide what to do with these images.

Reply:
Noted.